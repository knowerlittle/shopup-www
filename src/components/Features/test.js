import React from 'react';
import Feature from './';
import {
  shallow
} from 'enzyme';
import toJson from 'enzyme-to-json';


describe('Feature component tests', () => {
  test('<Feature /> renders correctly', () => {
    const tree = shallow( <
      Feature / >
    );
    expect(toJson(tree)).toMatchSnapshot();
  });
});