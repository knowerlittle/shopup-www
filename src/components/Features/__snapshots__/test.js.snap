// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feature component tests <Feature /> renders correctly 1`] = `
<Segment
  style={
    Object {
      'padding': '8em 0em',
    }
  }
  vertical={true}
>
  <Grid
    container={true}
    stackable={true}
    verticalAlign='middle'
  >
    <GridRow>
      <GridColumn
        width={8}
      >
        <Header
          as='h3'
          style={
            Object {
              'fontSize': '2em',
            }
          }
        >
          Lorem ipsum dolor sit, amet consectetur adipisicing elit
        </Header>
        <p
          style={
            Object {
              'fontSize': '1.33em',
            }
          }
        >
          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Blanditiis, saepe sapiente qui natus voluptates itaque quaerat magnam recusandae aliquid ut asperiores.
        </p>
        <Header
          as='h3'
          style={
            Object {
              'fontSize': '2em',
            }
          }
        >
          Lorem ipsum dolor sit, amet consectetur adipisicing elit
        </Header>
        <p
          style={
            Object {
              'fontSize': '1.33em',
            }
          }
        >
          Lorem ipsum dolor sit, amet consectetur adipisicing elit. Blanditiis, saepe sapiente qui natus voluptates itaque quaerat magnam recusandae.
        </p>
      </GridColumn>
      <GridColumn
        floated='right'
        width={6}
      >
        <Image
          as='img'
          bordered={true}
          rounded={true}
          size='large'
          src='http://via.placeholder.com/700x550'
          ui={true}
        />
      </GridColumn>
    </GridRow>
  </Grid>
</Segment>
`;
